### PR TITLE
Package GCP live Kubernetes visualizer

### PR DESCRIPTION
**Pull request by itskingori**
Google Cloud Platform live Kubernetes visualizer based off [saturnism/gcp-live-k8s-visualizer][1].

[1]: https://github.com/saturnism/gcp-live-k8s-visualizer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
...
+
+Google Cloud Platform live Kubernetes visualizer based off [saturnism/gcp-live-k8s-visualizer][1].
+
+## Usage
+
+```
+$ docker run \
+  --publish=8001:8001 \
+  --volume=$HOME/.kube:/home/visualizer/.kube:ro \
+  zappi/kubernetes-visualizer:<VERSION> proxy
```
Comment (adrianmoisey) [inline README.md]: I think `proxy` can be removed here

Comment (itskingori) [inline README.md]: True. Removed in [6fa00dd][1].

[1]: https://github.com/Intellection/docker-kubernetes-visualizer/pull/1/commits/6fa00dd7af34411318ad646c0650ecff780af669

File: README.md [hunk 2]
```diff
@@ -1,2 +1,21 @@
...
+  --publish=8001:8001 \
+  --volume=$HOME/.kube:/home/visualizer/.kube:ro \
+  zappi/kubernetes-visualizer:<VERSION>
+Starting to serve on [::]:8001
+```
+
+Once it's running, the visualizer should be available at `<public-address>:8001/static/`.
+
+Ps: Mounting the Kubernetes configuration from `$HOME` is ideal for local
+testing, but in Kubernetes you pod/container should have access to the service
```
Comment (adrianmoisey) [inline README.md]: `your` or `the` instead of `you`

Comment (itskingori) [inline README.md]: Fixed in [f436ed8][1].

[1]: https://github.com/Intellection/docker-kubernetes-visualizer/pull/1/commits/f436ed8c60e57acda015e54cbdf39659306a3455

**Pull request merged**